x-pack/auditbeat/module/system/sockets/_meta/fields.yml

```diff
@@ -0,0 +1,66 @@
+- name: sockets
+  type: group
+  description: >
+    `sockets` contains socket information.
+  release: experimental
+  fields:
+  - name: socket
+    type: array
+    description: >
+        Information about one or more sockets.
+    fields:
+    - name: family
+      type: keyword
+      example: ipv4
+      description: >
+        Address family.
+    - name: state
+      type: keyword
+      description: >
+        TCP state.
+    - name: direction
+      type: keyword
+      example: incoming
+      description: >
+        How the socket was initiated. Possible values are incoming, outgoing,
+        or listening.
```

Comment:
Right now only "inbound", "outbound" and "unknown" are specified in ECS. But I'd be fine with using "listening" there as well.

Reply:
The values come out of [Metricbeat's `system/socket` metricset](https://github.com/elastic/beats/blob/master/metricbeat/module/system/socket/listeners.go#L40). For now, I've changed to custom string values to be ECS compatible. We might want to consider changing Metricbeat as well.